From ca80554f4cf8b3e9111fb25938b6b82de17bdc55 Mon Sep 17 00:00:00 2001
From: John Kacur <jkacur@redhat.com>
Date: Sat, 12 Dec 2015 04:21:49 +0100
Subject: [PATCH 1/3] tuna: docs: Fix minor spelling error in the tuna man page

- Fix a grammar mistake.
- Fix the spelling of "across" in the tuna man page

Signed-off-by: John Kacur <jkacur@redhat.com>
---
 docs/tuna.8 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/tuna.8 b/docs/tuna.8
index 4864a9941bd1..a0a504cb13b9 100644
--- a/docs/tuna.8
+++ b/docs/tuna.8
@@ -5,7 +5,7 @@ tuna \- program for tuning running processes
 .B tuna
 [\fIOPTIONS\fR]
 .SH DESCRIPTION
-This manual page explains the \fBtuna\fR program. The program can be used to change the attributes of application and kernel threads. The \fBtuna\fR can operate on IRQs by name or number, and tasks or threads by process ID or command-line. CPUs and sets of CPUs can be specified by CPU or socket number. IRQ names and process command-lines can include wildcards.
+This manual page explains the \fBtuna\fR program. The program can be used to change the attributes of application and kernel threads. \fBtuna\fR can operate on IRQs by name or number, and tasks or threads by process ID or command-line. CPUs and sets of CPUs can be specified by CPU or socket number. IRQ names and process command-lines can include wildcards.
 .PP 
 \fBtuna\fP can change scheduling policy, scheduler priority and processor affinity for processes and process threads. \fBtuna\fR can also change the processor affinity for interrupts.
 When \fBtuna\fR is invoked without any options it starts up in its graphical interface mode. This manual page explains only the command\-line options for \fBtuna\fR
@@ -84,4 +84,4 @@ If for instance the Ethernet NICs have multiple queues for both receive and tran
 .TP
 .B tuna -S 2 -i -q 'eth*' -x
 
-Move everything off the CPUs in socket 2, then spread the IRQs for the Ethernet devices acrross those same CPUs.
+Move everything off the CPUs in socket 2, then spread the IRQs for the Ethernet devices across those same CPUs.
-- 
2.4.3

